### DISCUSSION TITLE
is popper.js an industry standard?

### DISCUSSION THREAD

**Discussion by slk333** [floating-ui/floating-ui#1212] [Q&A]
I'm about to use Popper.js in my project and I want to make sure there is no other native way that works just as well.
Also, how come there is no native solution for such an important UI concept that are popovers and tooltips?

**Comment by FezVrasta** [comment]
I'm not sure who decides something to be considered an *industry standard* but Popper adoption is widespread. Browsers don't support any native system to do the same.

**Comment by theo-staizen** [comment]
I would argue popper is as close to an industry standard as you can get.
Currently the only other positioning library that can compete with Popper in terms of features is [tether.io](http://tether.io).

Each have their own pros and cons. If you are starting fresh, I'd go with Popper

**Comment by slk333** [comment]
turns out Tippy.js suits my needs way better than Popper.js
I think you should put more emphasis on the fact that popper.js does not provide a ready-to-use solution for tooltips and dropdowns, as we jump in the ship and realize we might not need to reinvent the wheel and that Tippy.js provides what we actually need. (based on Popper.js)

**Reply by FezVrasta** [reply]
You mean something like this?
https://popper.js.org/docs/v2/tippy/
